test/e2e/pager.test.js

```diff
@@ -9,4 +9,83 @@ describe('vl-pager', async () => {
         return vlPagerPage.load();
     });
 
+    it('als ik kan op volgende klik dan zal de pager naar de volgende pagina gaan', async () => {
```

Comment:
Misschien ook nog ergens testen dat de getoonde pagina's opschuift als je naar een volgende pagina gaat

Reply:
zit reeds in de unit testen.